<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="agregarButton">
      <property name="text">
       <string>Agregar amigo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="archivoButton">
      <property name="text">
       <string>Enviar archivo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="descargasButton">
      <property name="text">
       <string>Aceptar archivos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="volverButton">
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>121</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enviarButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="mensajesTextEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Amigos:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
